manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml

```
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 0.11.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
```

```suggestion
  Scope: user
```
It's a `user` application and isn't `machine`.

---

It will fail again then.

---

We won't be able to accept the PR if it's set to `machine`.